WEEK1 SOLUTION/script1.py

```diff
@@ -0,0 +1,8 @@
+#check number is prime or not...
+n= int(input("Enter any number: "))
+for i in range(2,n):
+    if n%i==0:
+        print("{} is not prime number...!".format(n))
+        break
+else:
+    print("{} is prime number.".format(n))

```

Comment:
Do you know any other way to write same statement...?

Reply:
Yess.
Here that another way...
 print(f"{n} is  prime number.")